game/scenery_group.py

```diff
@@ -82,12 +80,10 @@ def make_scenery_groups(trigger_zones: Iterable[TriggerZone], blue: bool):
 
     @staticmethod
     def is_blue(zone: TriggerZone) -> bool:
+        """Blue in RGB is [0 Red], [0 Green], [1 Blue].  Ignore the fourth position: Transparency."""
```

Comment:
```suggestion
        # Blue in RGB is [0 Red], [0 Green], [1 Blue]. Ignore the fourth position: Transparency.
```

(this is just a comment explaining an implementation detail, not API documentation, which is what docstrings are for)